projects/plots/apptainer.def

```diff
@@ -44,3 +44,4 @@ apt-get clean
 cd /opt/aframe/projects/plots
 pip install -r requirements.txt --no-deps --no-cache-dir
 pip install -e . --no-deps --no-cache-dir
+pip install pytest
```

Comment:
Should probably just add this as a dev dep in the `pyproject.toml`

Reply:
Oh, yup, I thought it was already there. I think it needs to be in the apptainer file though - we have it for `train`, `export`, and `infer`.